addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -4677,7 +4677,165 @@ registry.DynamicSvg = SnippetOptionWidget.extend({
     },
 });
 
+/**
+* Allows to copy the anchor to clipboard and optionally rename the anchor.
+*/
+const CopyAnchorToClipboardAndNotifyWidget = SnippetOptionWidget.extend({
```

Comment:
Why was this moved to web_editor? Keep it in website, especially since it is only used there...

Reply:
@IT-Ideas do you remember why you moved that here? can I put it back in website?

Reply:
Hey! @bvr-odoo , I guess because I did not know about the snippet architecture and design at that time...